addons/web/models/base_document_layout.py

```diff
@@ -138,8 +138,10 @@ def _compute_preview(self):
                     'preview_css': preview_css,
                     'is_html_empty': is_html_empty,
                 })
+                return ir_ui_view, wizard_with_logo, preview_css, is_html_empty
             else:
                 wizard.preview = False
+                return wizard.preview
```

Comment:
Here it will crash if it goes to the super i think, maybe return a list of element and in the super check the len ?